rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -415,6 +415,7 @@ def attempt_connect(
         except requests.exceptions.RequestException:
             message = f'Failed to connect to {self.chain_name} node {node} at endpoint {rpc_endpoint}'  # noqa: E501
             log.warning(message)
+            self.failed_nodes.add(node.name)
```

Comment:
Also add a small comment explaining what this set is. Would you ever be able to reset this set/ remove somehting out of this so long as the app runs?

```suggestion
            self.failed_to_connect_nodes.add(node.name)
```

Reply:
At the moment no, we would need to restart. We can add additional logic maybe to re-check